### PR TITLE
Fixed #30567 -- Made WSGIHandler pass FileResponse.block_size to wsgi.file_wrapper.

### PR DESCRIPTION
**Pull request by domandinho**

### CODE REVIEW

File: tests/handlers/test_wsgi.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            'SERVER_PORT': '8000',
+            'wsgi.file_wrapper': lambda *args: DummyFileResponse(None, args[1] + 1),
+        }
+
+    def test_wsgi_block_size_passing(self):
+        sample_response = FileResponse()
+        sample_response.file_to_stream = iter(['sample_response'])
+        with mock.patch.object(WSGIHandler, 'get_response', return_value=sample_response):
+            response = WSGIHandler()(self._get_suspicious_environ(), lambda *a, **k: None)
+        assert response.value == FileResponse.block_size + 1
```
Comment (felixxm) [inline tests/handlers/test_wsgi.py]: Use `self.assertEqual()` instead of `assert ... == ...` .

**Comment by felixxm** [comment]
To be honest this test can be added as a part of `WSGITest.test_file_wrapper()`, e.g.
```diff
diff --git a/tests/wsgi/tests.py b/tests/wsgi/tests.py
index f8ee08e387..d1d721450d 100644
--- a/tests/wsgi/tests.py
+++ b/tests/wsgi/tests.py
@@ -51,7 +51,8 @@ class WSGITest(SimpleTestCase):
         FileResponse uses wsgi.file_wrapper.
         """
         class FileWrapper:
-            def __init__(self, filelike, blksize=8192):
+            def __init__(self, filelike, block_size=8192):
+                self.block_size = block_size
                 filelike.close()
         application = get_wsgi_application()
         environ = self.request_factory._base_environ(
@@ -67,6 +68,7 @@ class WSGITest(SimpleTestCase):
         response = application(environ, start_response)
         self.assertEqual(response_data['status'], '200 OK')
         self.assertIsInstance(response, FileWrapper)
+        self.assertEqual(response.block_size, 4096)
```

File: tests/handlers/test_wsgi.py (hunk 1 above)
Comment (domandinho) [reply tests/handlers/test_wsgi.py]: Thanks :) Done. Please take a look at this PR :)

**Comment by felixxm** [comment]
@domandinho Thanks! Welcome aboard :motor_boat:

**Pull request merged**